CDCgov/prime-simplereport
# upgrade packages with old subdependencies

# FRONTEND PULL REQUEST

## Related Issue

Upgrading dev dependencies that were using mismatched versions of `@testing-library/dom`. This was causing a bunch of `act` warnings in another PR that get fixed if we update these dependencies so that they use the same subdependency version of `@testing-library/dom`.

## Testing
- Run storybook locally and make sure it still works
- Make sure the relevant eslint plugin still fixes errors. I plugged in a violation to [this rule](https://github.com/testing-library/eslint-plugin-jest-dom/blob/main/docs/rules/prefer-to-have-value.md) into a test file and made sure it fixed on save still.

## Screenshots / Demos


### After
https://github.com/CDCgov/prime-simplereport/assets/29645040/ef23b15c-d114-4b0b-aaa8-ba06c9814d5a
